tests/Sniffs/Namespaces/data/caseInsensitiveUse.php

```diff
@@ -11,7 +11,9 @@
 new BAR();
 
 /** @var boo */
-/** @BOO */
+/**
```

Comment:
Please add special annotation for your test case and don't change existing test cases.